### PR TITLE
Update Statistics.md

### PR DESCRIPTION
**Pull request by vuhlmann**
Modified content to address most comments and fixed a few typos

### CODE REVIEW

**Comment by vuhlmann** [comment]
The reference list (references.bib) must be updated to include the following:

@ARTICLE{Lever2017-pca,
  title    = "Principal component analysis",
  author   = "Lever, Jake and Krzywinski, Martin and Altman, Naomi",
  journal  = "Nat. Methods",
  volume   =  14,
  pages    = "641--642",
  month    =  july,
  year     =  2017,
  language = "en"
}

**Comment by bethac07** [comment]
Thanks Virginie! Do you want to add that reference to the citation file, or would you like me to?

**Comment by vuhlmann** [comment]
I tried to edit this file myself but miserably failed for some reason. Let me try again!

**Review by bethac07 — APPROVED** [review]
